cmd/bosun/conf/conf.go

```diff
@@ -903,6 +903,14 @@ func (c *Conf) findAllDependencies(a *Alert) {
 	var walkNotifications = func(n *Notifications) {
 		for _, l := range n.Lookups {
 			addIfUnique(&l.ConfItem)
+			// if we see a lookup here, assume all leaf nodes are notifications
+			for _, entry := range l.Entries {
+				for _, lookupNot := range entry.Values {
+					if found, ok := c.Notifications[lookupNot]; ok {
```

Comment:
I don't think so. We only got here because warnNotification was set to a lookup. I don't think that lookup could return a macro and still work in that context.